<launch>
    <rosparam command="load" file="$(find fastlio)/config/localize.yaml" />
    <node pkg="fastlio" type="localizer_node" name="localizer_node" output="screen"/>
    <!-- <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find fastlio)/rviz/localize.rviz" /> -->
</launch>